lib/repo/git_repository.rb

```diff
@@ -74,9 +78,12 @@ def self.create(connect_string)
       index = repo.index
       index.add(path: 'README.md', oid: oid, mode: 0100644)
       index.write
-      Rugged::Commit.create(repo,
-                            commit_options(repo, 'Markus',
-                                           'Initial commit and add readme.'))
+      Rugged::Commit.create(
+	repo,
+	commit_options(
+		repo,
+		'Markus',
+	        'Initial readme commit.'))
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Tab detected.